lib/infrastructure/utils.js

```diff
@@ -56,9 +56,10 @@ module.exports = function Utils() {
 
     function _exec(command, cwd) {
         return new Promise(function(resolve, reject) {
-            exec(command, {cwd: cwd}, function(error, stdout) {
+            exec(command, {cwd: cwd}, function(error, stdout, stderr) {
                 if(error) {
-                    logger.log('Error during "{0}" in "{1}". Output was: {2}'.format(command, cwd, stdout));
+                    logger.log('Error during "{0}" in "{1}".');
```

Comment:
yea, i'm sorry, getting sleep & sloppy.  I am behind a proxy, and git wasn't configured to use the proxy.  I wasn't getting any error output, so perhaps it's necessary to output stdout & stderr.  I'll fix the formatter and resubmit